src/utils/files.ts

```
@@ -16,3 +16,23 @@
 
   return filePaths
 }
+
+
+export async function getFilesInDirectoryRecursive(dir: string): Promise<string[]> {
```

ESLint

[error] More than 1 blank line not allowed.

(@stylistic/no-multiple-empty-lines)